cln/cln_client.go

```diff
@@ -121,10 +116,11 @@ func (c *ClnClient) OpenChannel(req *lightning.OpenChannelRequest) (*wire.OutPoi
 		!req.IsPrivate,
 		minConfs,
 		glightning.NewMsat(0),
-		minDepth,
```

Comment:
Same for the mindepth parameter. We can't remove that, because we need it for zero conf channels.